Applications/Spire/Include/Spire/OrderImbalanceIndicator/OrderImbalanceIndicatorContextMenu.hpp

```diff
@@ -0,0 +1,87 @@
+#ifndef SPIRE_ORDER_IMBALANCE_INDICATOR_CONTEXT_MENU_HPP
+#define SPIRE_ORDER_IMBALANCE_INDICATOR_CONTEXT_MENU_HPP
+#include <QMenu>
+#include "Spire/Spire/Spire.hpp"
+
+namespace Spire {
+
+  //! Displays a context menu for the order imbalance indicator window.
+  class OrderImbalanceIndicatorContextMenu : public QMenu {
+    public:
+
+      //! Signals that a menu option was selected.
+      using SelectedSignal = Signal<void ()>;
+
+      //! Signals that a column check box was toggled.
+      /*
+        \param is_checked True if the check box is in the checked state.
+      */
+      using ToggledSignal = Signal<void (bool is_checked)>;
+
+      //! Constructs an OrderImbalanceINdicatorContextMenu.
+      /*
+        \param parent The parent widget.
+      */
+      OrderImbalanceIndicatorContextMenu(QWidget* parent = nullptr);
```

Comment:
What will happen in `parent` is `nullptr`? Is it a valid use case?
Regardless, unless you actually need implicit conversion from `QWidget` pointers to `OrderImbalanceIndicatorContextMenu`, make this constructor `explicit`.

Reply:
I've marked it explicit. The default parameter here is a Qt convention that we're copied for Spire.